cli/cli.go

```diff
@@ -284,6 +290,31 @@ func fetchIndex() {
 	}
 }
 
+func installFromVersionFile() {
+	blob, err := ioutil.ReadFile(VERSION_FILE)
+	if err != nil {
+		fmtc.Println("Cannot read %s", VERSION_FILE)
+		exit(1)
+	}
+
+	version, err := readFirstWord(string(blob))
+	if err != nil {
+		fmtc.Println("Cannot find version in %s", VERSION_FILE)
+		exit(1)
+	}
+
+	fmtc.Println("Installing version %s from %s", version, VERSION_FILE)
+	installCommand(version)
+}
+
+func readFirstWord(body string) (string, error) {
+	matches := regexp.MustCompile(`^\s*(\S+)\s*`).FindStringSubmatch(body)
+	if len(matches) < 2 {
+		return "", errors.New("I have no words!")
```

Comment:
I don't understand why you call it "unused". I know only one way of making errors: `errors.New("Some text")`. So, I see only one choice: make text boring or funny.
If there is a way to return an error without text, please let me know.